torchci/components/NavBar.tsx

```diff
@@ -50,8 +50,8 @@ function NavBar() {
       href: "/torchbench/userbenchmark",
     },
     {
-      name: "Triton Compile",
```

Comment:
Should we make this more generic since its more than just triton kernels

Something like Operator Benchmarks?